frontend/src/metabase/query_builder/components/expressions/ExpressionEditorSuggestions.styled.jsx

```diff
@@ -0,0 +1,24 @@
+import styled from "styled-components";
```

Comment:
👍 for styles separation. I also like when it is separate which makes it easier to read component logic. We might want to consider to stick with it